core/src/tile/tileManager.cpp

```diff
@@ -237,6 +238,9 @@ void TileManager::updateTileSet(TileSet& _tileSet, const ViewState& _view,
                 entry.task = _tileSet.source->createTask(visTileId);
                 enqueueTask(_tileSet, visTileId, _view);
                 m_tilesInProgress++;
+            } else if (!entry.needsLoading() && !entry.isReady()) {
+                // tile not generated yet!
+                m_tilesInGeneration++;
             }
```

Comment:
I wanted to explicitly differentiate it. Makes understanding tilemanager code better.